### PR TITLE
Feature/177201996/action log servlet

### PR DESCRIPTION
**Pull request by alex-melnik**
https://www.pivotaltracker.com/story/show/177201996

### CODE REVIEW

File: packages/hiro-graph-client/index.d.ts [hunk 1]
```diff
@@ -448,6 +448,94 @@ export interface VariablesServlet {
...
+
+export interface KIEvent extends BaseEvent {
+  kiId: string;
+  kiName: string;
+}
+
+export interface TeachingSessionEvent extends BaseEvent {
+  sessionId: string;
+  sessionName: string;
+  issueId: string;
```
Comment (bpneal) [inline packages/hiro-graph-client/index.d.ts:486]: shall we use issueId or taskId ?

**Review by bpneal — APPROVED** [review]

File: packages/hiro-graph-client/index.d.ts (hunk 1 above)
Comment (alex-melnik) [reply packages/hiro-graph-client/index.d.ts:485]: I saw `issueId` here: https://www.pivotaltracker.com/story/show/177142344
Should we change it to `taskId` ?

Comment (nshostak) [reply packages/hiro-graph-client/index.d.ts:485]: it's issueId in aggregating task https://www.pivotaltracker.com/story/show/177142344
do I need to change it?

Comment (bpneal) [reply packages/hiro-graph-client/index.d.ts]: Please can we change to taskId, I've updated all tickets. Sorry this was my mistake

**Review by bpneal — APPROVED** [review]

File: packages/hiro-graph-client/index.d.ts [hunk 2]
```diff
@@ -448,6 +448,97 @@ export interface VariablesServlet {
...
+  | 'OwnTeachingSession'
+  | 'StartTeachingSession'
+  | 'ContinueTeachingSession'
+  | 'StartConversionSession'
+  | 'ContinueConversionSession'
+  | 'AssignKI';
+
+export interface BaseEvent<T extends ActionLogMeta> {
+  action: ActionLogType;
+  instanceId: string;
```
Comment (nshostak) [inline packages/hiro-graph-client/index.d.ts:469]: should it be `instance_id`? as Vitaly mentioned

**Review by yury-klika — APPROVED** [review]

**Review by nshostak — APPROVED** [review]

File: packages/hiro-graph-client/index.d.ts (hunk 2 above)
Comment (bpneal) [reply packages/hiro-graph-client/index.d.ts:469]: no, I've checked, it should be `instanceId`

File: packages/hiro-graph-client/index.d.ts [hunk 3]
```diff
@@ -448,6 +448,97 @@ export interface VariablesServlet {
...
+export interface TeachingSessionEventMeta {
+  sessionId: string;
+  sessionName: string;
+  taskId: string;
+  desktopRole: string;
+}
+
+export interface DeployKIEventMeta {
+  knowledgePoolId: string;
+  knowledgePoolName: string;
```
Comment (nshostak) [inline packages/hiro-graph-client/index.d.ts:491]: `kiId` and `kiName` are needed here as well

**Review by prapakou — APPROVED** [review]

File: packages/hiro-graph-client/index.d.ts (hunk 3 above)
Comment (alex-melnik) [reply packages/hiro-graph-client/index.d.ts:491]: Thanks, done!

**Pull request merged**